src/conn/conn_sweep.c

```diff
@@ -350,6 +355,9 @@ __sweep_server(void *arg)
 
 		if (dead_handles > 0)
 			WT_ERR(__sweep_remove_handles(session));
+
+		/* Remember last sweep time. */
+		last = now;
```

Comment:
@agorrod , I think we had a bug that went unnoticed. We were never updating the `last` time sweep ran. I had noticed in T2 data from Atlas that we were running sweep every 2 seconds, most likely because of this.

Reply:
![screen shot 2019-01-31 at 13 53 02](https://user-images.githubusercontent.com/19491057/52027454-9bf29d00-255f-11e9-8a23-eb45c8497451.png)

For instance, those connection sweeps happen every 2 seconds.